frontend/server/src/Controllers/Session.php

```diff
@@ -46,7 +46,7 @@ public static function getSessionManagerInstance(): \OmegaUp\SessionManager {
     public static function getFacebookLoginUrl(): string {
         $scopedFacebook = new ScopedFacebook();
         return $scopedFacebook->facebook->getAuthorizationUrl([
-            'scope' => ['email'],
+        'scope' => ['email'],
```

Comment:
```suggestion
            'scope' => ['email'],
```

same as in the other change: can the whitespace changes be minimized?

Reply:
Sure! But i didn't touch this piece of code...
Should i didn't run this command ./stuff/lint.sh ?

Reply:
your editor probably modified it by itself without you noticing and `./stuff/lint.sh` couldn't fix it.

Reply:
Ohh okayy! But I had off the formatting of my editor. I noticed sometimes when i run this command these space problems occurred while sometimes not.